vignettes/transition_disruption_metric.Rmd

```diff
@@ -0,0 +1,552 @@
+---
+title: "Transition Disruption Metric"
+author: "Jackson Hoffart"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{transition_disruption_metric}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library(dplyr)
+library(ggplot2)
+library(PACTA.analysis)
+```
+
+# Introduction
+
+PACTA offers many different metrics, each with their own strengths and weaknesses. Most of the metrics rely heavily on the use of forward-looking data. This data, which estimates the production of physical assets, is reliably predictable to about five years into the future. 
+
+The other key component to these metrics is a climate scenario, which projects how production *should* shift, to mediate climate change. There are, however, certain climate scenarios which deliberately offset the bulk of their ambition to the later years in the scenario (sometimes 20+ years into the future). A "delayed response" scenario, such as the  Inevitable Policy Response (IPR) Scenario, assumes that future production of most sectors follow current trends (i.e. business as usual), before inevitably resulting in a massive and disruptive transition in the long-term. Since the interesting part of these scenarios is often positioned well beyond the 5 year time-horizon of the data, most usual PACTA metrics (eg. Volume Trajectory) are ill-suited to provide useful insights. 
+
+The purpose of these "delayed response" scenarios is to estimate the risks associated with a "worst-case scenario". With such a scenario, we might want to measure how **disruptive** a climate **transition** could be to a company or portfolio.  
+
+To help provide insights into this question, we propose the Transition Disruption Metric (TDM). This metric aims to provide useful insights in the medium- to long- term (ie. time-horizons greater than 5 years).
+
+# Derivation
```

Comment:
Before I (a generic reader) read the rest of the article I need an overview of what to expect. Without that tool I can't navigate the article; I don't know where I'm going, nor if I'm close to the end. 

It's okay to draft your ideas as they come to mind. But you need to organize them in a meaningful way before the final version. Imagine you ask me how to get my place. You're only interested in the fastest or more direct way, not in all the detours I might to as I run my errands on my way home.